app/assets/stylesheets/bootstrap_and_overrides.css

```diff
@@ -62,24 +68,16 @@
     padding-top: 50px;
   }
 
-  body
-  {
+  body {
     font-family: 'Hind', sans-serif;
     font-size: 16px;
-    padding-top: 80px;
     color: #555;
     background-image: url('http://i.imgur.com/WtYG1kl.png');
     background-attachment: fixed;
-    background-repeat: no-repeat;
```

Comment:
@ryanflach Because we're using `background-size: cover` and `background-attachment: fixed`, there's no need to set the background to `no-repeat`, since the other 2 CSS lines make it so the background covers the entire screen and doesn't scroll. The added benefit of removing `no-repeat` is that I'm pretty sure it will make the background repeat **only on mobile**, because currently if you look at our site on mobile, the background doesn't cover the whole screen.